### PR TITLE
Add tg3 package to publishing script

### PR DESCRIPTION
**Pull request by maurolacy**

### CODE REVIEW

File: scripts/publish.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@ set -o errexit -o nounset -o pipefail
 command -v shellcheck >/dev/null && shellcheck "$0"
 
 # These are imported by other packages - wait 30 seconds between each as they have linear dependencies
-BASE_CRATES="packages/bindings packages/bindings-test packages/tg4 packages/utils contracts/tg4-engagement contracts/tg4-stake contracts/tg4-mixer packages/voting-contract"
+BASE_CRATES="packages/bindings packages/bindings-test packages/tg4 packages/utils ontracts/tg4-engagement contracts/tg4-stake contracts/tg4-mixer packages/tg3 packages/voting-contract"
```
Comment (ueco-jb) [inline scripts/publish.sh:6]:
```suggestion
BASE_CRATES="packages/bindings packages/bindings-test packages/tg4 packages/utils contracts/tg4-engagement contracts/tg4-stake contracts/tg4-mixer packages/tg3 packages/voting-contract"
```

**Review by ueco-jb — APPROVED** [review]
typo

**Comment by ueco-jb** [comment]
Oh, so cocky of you to not wait for review!

**Comment by maurolacy** [comment]
I see... was pretty sure this was correct. Good eye, will fix the typo in a follow-up.

**Pull request merged**